.github/workflows/unit.yaml

```diff
@@ -79,5 +79,5 @@ jobs:
         if: startsWith(github.ref, 'refs/tags')
         uses: pypa/gh-action-pypi-publish@master
         with:
-          user: __token__
+          user: adiroiban
```

Comment:
I think that you should have `__token__` here.
I have created an API token.
I have not saved my password :)